frontend/kick-covid-19/src/pages/Posts.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import { useParams } from "react-router-dom";
+
+function Posts() {
+  const { id } = useParams();
+
+  return <h1>Post {id}</h1>;
+}
+
+export default Posts;
```

Comment:
also, if you didn't want to use hooks just for this, react router comes with a function (technically a higher order component) called `withRouter` that you can call with your component as an argument (e.g. `export default withRouter(Posts)` instead of `export default Posts`).  `withRouter` will pass  the params to your component via props in that case.  I prefer the hook but just figured I'd mention it 😊